controllers/userController.js

```diff
@@ -13,3 +13,16 @@ export async function getUser(req, res) {
     return res.sendStatus(500)
   }
 }
+
+export async function getUserBySearch(req, res){
```

Comment:
change the name to getUserByUsername. The name of function shouldn't be directly related to the front end.